vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/RealTunnelling.java

```diff
@@ -0,0 +1,46 @@
+package vpnrouter.core.infrastructure.shell;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import vpnrouter.core.service.client.client.Tunnelling;
+
+import java.util.Set;
+
+import static java.util.stream.Collectors.joining;
+
+@Primary
+@Component
+@Profile("prod")
+@RequiredArgsConstructor
+public class RealTunnelling implements Tunnelling {
+
+    private static final int TABLE = 7373;
+
+    private final NetworkProperties networkProperties;
+    private final ShellExecutor shellExecutor;
+
+    @Override
+    public void switchOnOnlyFor(Set<String> ipAddresses) {
+        shellExecutor.execute(
+                buildResetTableCommand() + buildSwitchOffCommand() + buildSwitchOnCommand(ipAddresses)
```

Comment:
shell commands seem so complicated to me 😱

Reply:
Then you can focus on Java code :)